src/ipa_pc/data_structures.rs

```diff
@@ -263,7 +262,7 @@ pub struct BatchProof<G: AffineCurve> {
     pub proof: Proof<G>,
 
     /// Commitment of the h(X) polynomial
-    pub batch_commitment: G, 
+    pub batch_commitment: Vec<G>,
 
     /// Values: v_i = p_i(x), where x is fresh random challenge
```

Comment:
let us change the comment here to
"Values: v_i = p(x_i), where the query points x_i are not necessarily distinct."

Reply:
Done